prow/config.yaml

```diff
@@ -198,6 +198,38 @@ presubmits:
         secret:
           secretName: istio-e2e-rbac-kubeconfig
     run_after_success:
+    - name: istio-unit-test
+      agent: kubernetes
+      context: prow/istio-unit-test.sh
+      rerun_command: "/test istio-unit-test"
+      trigger: "(?m)^/test (istio-unit-test)?,?(\\s+|$)"
+      max_concurrency: 8
+      spec:
+        containers:
+        - image: gcr.io/istio-testing/prowbazel:0.3.5
+          args:
+          - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+          - "--clean"
+          - "--timeout=60"
+          # Bazel needs privileged mode in order to sandbox builds.
+          securityContext:
+            privileged: true
+          env:
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/service-account/service-account.json
+          volumeMounts:
+          - name: service
+            mountPath: /etc/service-account
+            readOnly: true
+          - name: kubeconfig
```

Comment:
Do we still need a kubeconfig for unit tests ?

Reply:
Yes, afaik pilot, security unit tests require them.

Reply:
You are right. I kept it since `setup_and_export_git_sha` in `lib.sh` uses it but then it is just exporting the directory and no unit tests access it so I have removed it in latest commit